bin/migrate-oats-data/documents/document_source_update.py

```diff
@@ -124,6 +125,9 @@ def _get_document_source(data):
     alcs_source = data.get("source", "")
     if alcs_source != 'oats_etl' and alcs_source:
         return alcs_source
-    # Source is ALC/APP/LG as per 1558
-    else:
-        return source
\ No newline at end of file
+    elif source:
+        source = str(
+                OatsToAlcsDocumentSourceCode[source].value
```

Comment:
Is it possible that the source is not in the oats enum, or are there only those 3 values in the documents table?

Reply:
oats_document_source_codes table only has 3 entries and the column is not null-able in the documents table